### ISSUE TITLE
Kibi Timeline Duplication

### ISSUE THREAD

**Issue by giorgio94c** [sirensolutions/kibi_timeline_vis#42]
From [kibi-users forum](https://groups.google.com/a/siren.solutions/forum/#!topic/kibi-users/CQv3kZBrrj4)

> So I built myself my first timeline today, and I was wondering why nothing had any labels. Turns out .raw labels don't seem to be supported, the field just comes up as "" and the darn thing is, when I use the regular analyzed form of the field, I end up getting tons of duplicates. Is there anything I can do to fix this, or is this just how it is?

Copy of https://github.com/sirensolutions/kibi-internal/issues/1465

**Comment by giorgiocasati** [comment]
Can we have some more information about this?

**Comment by scampi** [comment]
I was thinking I can try this issue and you can take care of the other two you are assigned to.

**Comment by scampi** [comment]
I managed to reproduce the issue with the `.raw` field using the following setup.

Mapping:

```sh
$ curl -XPUT 'localhost:9200/weather' -d '
{
  "mappings": {
    "Weather": {
      "properties": {
        "forecast": {
          "type": "string", "index": "analyzed",
          "fields": {
            "raw": {
              "type":  "string",
              "index": "not_analyzed"
            }
          }
        }
      }
    }
  }
}'
```

Data:

```sh
$ curl -XPOST 'localhost:9200/weather/Weather' -d '
{
  "date": "2016-09-06T15:37+00:00",
  "forecast": "cloudy, drizzle"
}
'

$ curl -XPOST 'localhost:9200/weather/Weather' -d '
{
  "date": "2016-09-06T15:40+00:00",
  "forecast": "cloudy, shower"
}
'
```

Visualization:

```json
[
  {
    "_id": "Timeline",
    "_type": "visualization",
    "_source": {
      "title": "Timeline",
      "visState": "{\"title\":\"Timeline\",\"type\":\"kibi_timeline\",\"params\":{\"groups\":[{\"id\":5000,\"groupLabel\":\"Weather\",\"color\":\"#6f87d8\",\"endField\":null,\"labelField\":\"forecast\",\"startField\":\"date\",\"savedSearchId\":\"Weather\",\"indexPatternId\":\"weather\"}],\"groupsOnSeparateLevels\":false},\"aggs\":[],\"listeners\":{}}",
      "uiStateJSON": "{}",
      "description": "",
      "version": 1,
      "kibanaSavedObjectMeta": {
        "searchSourceJSON": "{\"query\":{\"query_string\":{\"query\":\"*\",\"analyze_wildcard\":true}},\"filter\":[]}"
      }
    }
  }
]
```

**Comment by scampi** [comment]
I don't see how to differentiate between multifields so that we wouldn't show them in the dropdown menu. Maybe a good enough solution is to have some default label when the selected event label is empty, e.g., `No value for field XXX`.

**Comment by SergeyBondarenko** [comment]
@scampi, now if the label field is empty the object label on the timeline is `N/A`:
![screenshot from 2017-02-06 14-31-10](https://cloud.githubusercontent.com/assets/5389745/22648923/08db5406-ec79-11e6-8104-e18ab823c816.png)
The multi-fields feature implemented in this PR https://github.com/sirensolutions/kibi_timeline_vis/pull/117

**Comment by SergeyBondarenko** [comment]
I can't reproduce this duplication issue.

**Comment by scampi** [comment]
fixed with #84
